FirebaseRemoteConfig/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+$ v4.4.8
+- [changed] Update OCMock version dependency to v3.5 in Remote Config podspec.
```

Comment:
CHANGELOG not necessary since the change only impacts unit tests and not product.

Reply:
sg.